calc/src/App.js

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import Buttons from "./Components/Buttons";
+import "./App.css";
+
+class App extends React.Component {
+  state = {
+    inputA: "0",
+    inputB: "0",
+    operator: "",
+    complete: false,
+  };
+
+  numberButton = (value) => {
+    if (this.state.inputA === "0" || this.state.complete === true) {
+      this.setState({ inputA: value, complete: false });
+    } else {
+      this.setState((prevState) => ({
+        inputA: prevState.inputA + value,
+      }));
+    }
+  };
+
+  operators = (value) => {
+    const arr = ["+", "-", "*", "÷"];
+    for (let i = 0; i < arr.length; i++) {
+      if (value === arr[i]) {
+        this.setState((prevState) => ({ inputB: prevState.inputA }));
+        this.setState({ operator: arr[i] });
+        this.setState({ inputA: "0" });
+      }
+    }
+  };
+
+  equals = () => {
+    const { inputA, inputB, operator } = this.state;
+    console.log(this.state);
+    this.setState({ complete: true });
+    if (operator === "+") {
```

Comment:
A switch statement would be better suited for this conditional. 
switch(operation) {
 case: "+":